repos/system_upgrade/common/models/trackedfiles.py

```diff
@@ -0,0 +1,60 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class FileInfo(Model):
+    """
+    Various data about a file.
+
+    This model is not supposed to be used as a message directly.
+    See e.g. :class:`TrackedSourceFilesInfo` instead.
+    """
+    topic = SystemInfoTopic
+
+    path = fields.String()
+    """
+    Canonical path to the file.
+    """
+
+    exists = fields.Boolean()
+    """
+    True if the file is present on the system.
+    """
+
+    rpm_name = fields.String(default="")
+    """
+    Name of the rpm that owns the file. Otherwise empty string if not owned
+    by any rpm.
+    """
+
+    # NOTE(pstodulk): I have been thinking about the "state"/"modified" field
+    # instead. Which could contain enum list, where could be specified what has
+    # been changed (checksum, type, owner, ...). But currently we do not have
+    # use cases for that and do not want to implement it now. So starting simply
+    # with this one.
+    is_modified = fields.Boolean()
+    """
+    True if the checksum of the file has been changed (includes the missing state).
+
+    The field is valid only for a file tracked by rpm - excluding ghost files.
+    In such a case the value is always false.
+    """
+
+
+class TrackedSourceFilesInfo(Model):
```

Comment:
@oamg/developers  we have discussed the name is confusing/amgiguous/no so clear. Let's put into this thread proposals about a different name.

For a context, providing reasoning behind the current (`TrackedSourceFilesInfo`) name:
* Tracked - it's targetting only files we explecitely marked (now in the code..) to track
* Source - it needs to be clear that information here is about the `source` system (that's terminology we use for the original system, used also in other existing models)
  * To be clear that info here does not have to be valid e.g. ager the upgrade transaction.
  * Make our way clear in case we would like to introduce in future "Target" version of this model
* FilesInfo - ...

My first additional proposal into the plenum (not so original... :D):
* TrackedFilesInfoSource

Reply:
I think maybe the original ``TrackedSourceFilesInfo`` sounds the best and makes the most sense after all.  Or maybe after stretching the grammar a little ...

- What are we interested in? => ``FilesInfo``
- File info from what files? -> only tracked! => ``TrackedFilesInfo``
- But where is being tracked from? -> source! => ``SourceTrackedFilesInfo`` (this sounds like the source is the one doing the tracking though) or more explicit ``TrackedFilesInfoFromSource`` (``TrackedFilesInfoOfSource`` ?)

Reply:
@dkubek thanks for the thinking about this. Basically we have very similar opinion on this. I prefer to keep the `Source` as a suffix in this case to be more compatible with @matejmatuska`s modesl for systemd services. However, I want to omit `From` and `Of` substrings in the name to keep it shorter and also due to consistency, as we do not use them neither.

We should write some best practices, tips .... for model names. It could help people with orientation in the set of models.